### DISCUSSION TITLE
Controllers custon onChange function

### DISCUSSION THREAD

**Discussion by jzrskc** [react-hook-form/react-hook-form#2724] [Q&A] [answered]
What is the best approach for the custom onChange function in Controllers renderer? I want to archive to call another function and at the same time default RFH onChange function to be called.

[code](https://codesandbox.io/s/distracted-jennings-4hbm0?file=/App.js)

**Comment by bluebill1049** [comment, marked as answer]
that looks right to me :).

**Reply by jzrskc** [reply]
Custom function _onChange is never called?

**Reply by bluebill1049** [reply]
you have overwrite it:
```
onChange={(val) => {
              _onChange(val);
              prop.onChange(val);
            }}
            {...prop} // here
```
